crates/common/src/ledger.rs

```diff
@@ -880,7 +880,17 @@ impl ChronicleOperation {
                 if let Some(graph) = json_ld.get_mut("@graph").and_then(|g| g.as_array_mut()) {
                     graph
                         .iter_mut()
-                        .map(|entry| entry.format_context_data())
+                        .map(|entry| {
```

Comment:
Duplicating the logic of `EvaluationContext::format_context_data` here to avoid owning each `entry`. Meanwhile, `EvaluationContext::format_context_data` allows us to clearly test how we're transforming the compacted prov model graph data with a view to experimentation down the line.